src/azure-cli/azure/cli/command_modules/backup/_params.py

```diff
@@ -153,3 +207,14 @@ def load_arguments(self, _):
 
     with self.argument_context('backup job wait') as c:
         c.argument('timeout', type=int, help='Maximum time, in seconds, to wait before aborting.')
+
+    with self.argument_context('backup recoveryconfig') as c:
```

Comment:
why are the parameters distributed in 2 groups?

Reply:
Optional ones are in second group. Can be merged as well provided no other sub command for recoveryconfig is needed.